pkg/kubelet/container/testing/fake_runtime.go

```diff
@@ -81,6 +81,12 @@ type FakeVersion struct {
 	Version string
 }
 
+func snapshot(imageList []kubecontainer.Image) []kubecontainer.Image {
```

Comment:
nit: it's a style preference of mine to define functions below the calling function so the file can be read top to bottom. would you mind moving this below?

Reply:
(that is, if we keep snapshot)

Reply:
thanks,i weill fix this,if we keep snapshot, :)

Reply:
hm this seems to have sneaked up again

Reply:
oh,i am sorry,fixed

> hm this seems to have sneaked up again

oh,i am sorry,fixed